recipes/y-py/build.sh

```diff
@@ -0,0 +1,7 @@
+curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
ah, this is installing rust? we should use the conda-forge supplied rust

Reply:
But the package needs nightly Rust.